src/components/progress-tracker/progress-tracker.component.js

```diff
@@ -49,6 +54,13 @@ const ProgressTracker = ({
     }
   }, [barRef, orientation]);
 
+  if (!isDeprecationWarningTriggered && isVertical) {
+    isDeprecationWarningTriggered = true;
+    Logger.deprecate(
+      "The `isVertical` prop of `ProgressTracker` is now deprecated."
+    );
+  }
+
```

Comment:
```suggestion
  if (!isDeprecationWarningTriggered && orientation) {
    isDeprecationWarningTriggered = true;
    Logger.deprecate(
      "The `orientation` prop of `ProgressTracker` is now deprecated."
    );
  }
```
It's the `orientation` prop were deprecating here. This will also need a unit test to check the warning is displayed in the console when the prop is provided. See this [similar example ](https://github.com/Sage/carbon/blob/9b29fd1c9245932d0a07b2378b3e774b7c1634eb/src/components/card/card.spec.tsx#L210) with `Card` for how to achieve this.